src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -31,8 +35,24 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping("/")
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id,
```

Comment:
just FYI

you may also validate such things as @PathVariable, @RequestParams etc. It looks like:
```
@PathVariable @Positive Long id
```

In order to be able to perform such validation you need to annotate your controller class with `@Validated` annotation

again, this is just FYI - apply it or not :)